cli/command/formatter/image_test.go

```diff
@@ -76,6 +76,11 @@ func TestImageContext(t *testing.T) {
 				i: types.ImageSummary{SharedSize: 5000, VirtualSize: 20000},
 			}, "15kB", ctx.UniqueSize,
 		},
+		{
+			imageContext{
+				i: types.ImageSummary{Created: -62135596800},
+			}, "", ctx.CreatedSince,
```

Comment:
Can we add also a test with `"N/A"`? And maybe group all the `Created` tests together?

Reply:
Hello @silvin-lubecki!

What I did was to add `N/A` tests to the whole output tests, inside `TestImageContextWrite` function. Also, since all the other tests are mixed, I didn't see how grouping just a couple of them would make sense (I believe that grouping everything would be better).

What do u think?